api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -232,7 +233,7 @@ private Workspace createWorkspaceImpl(Workspace workspace) {
         workbenchConfigProvider.get().billing.freeTierBillingAccountName());
     setDbWorkspaceFields(dbWorkspace, user, workspaceId, fcWorkspace, now);
 
-    setLiveCdrVersionId(dbWorkspace, workspace.getCdrVersionId());
```

Comment:
`setLiveCdrVersion()` did two things: query DB for a DbCdrVersion and assign it to a DbWorkspace.

Now we need the DbCdrVersion before the DbWorkspace exists.  This is also cleaner.